classes/class-wc-connect-service-settings-store.php

```diff
@@ -160,6 +160,67 @@ public function set_preferred_paper_size( $size ) {
 			return WC_Connect_Options::update_option( 'paper_size', $size );
 		}
 
+		/**
+		 * Gets post-label-printing notification settings.
+		 *
+		 * @return array Array of boolean values, indexed by setting names.
+		 */
+		public function get_post_print_notification_settings() {
+			$defaults = array(
+				'mark_order_complete_and_notify_customer' => false,
+				'notify_customer_with_shipment_details'   => false,
+			);
+
+			$settings = WC_Connect_Options::get_option( 'post_print_notification_settings', $defaults );
+
+			return array_merge( $defaults, $settings );
+		}
+
+		/**
+		 * Updates post-label-printing notification settings.
+		 *
+		 * @param string $name    Name of the setting to enable/disable.
+		 * @param bool   $enabled Whether the setting should be enabled.
+		 *
+		 * @return true|WP_Error WP_Error if an error occurred, `true` otherwise.
+		 */
+		public function set_post_print_notification_setting( $name, $enabled ) {
+			$allowed_names = array(
```

Comment:
We can't trust the user input and I like how you add the check here instead of the controller's level.